src/routes/_home/tags/demo-page/demo-page.style.module.scss

```
@@ -64,30 +64,29 @@
-    color: var(--color-white);
-    padding: 0.6em 2em;
-    display: inline-flex;
-    margin: 1em 0;
-    margin-bottom: 1.5em;
-
-    font-weight: 500;
-  }
-  &Text {
-    display: block;
-  }
+  font-weight: 500;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)